scripts/visualize_gwas_distribution.py

```diff
@@ -0,0 +1,116 @@
+"""
+2016 Gregory Way
+scripts/visualize_genomic_elements.py
+
+Description:
+Summarizes the location of genomic elements across TADs
+
+Usage:
+Is called by 'scripts/visualize.sh' which is run inside of
+'scripts/run_pipeline.sh'. This particular script will output the location
+of genomic elements in a given input TAD
+
+      python scripts/visualize_gwas_distribution.py
+
+Output:
+Two .pdf plots in "figures/" describing the GWAS signal distribution across
+hESC and IMR90 TADs
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from tad_util.util import load_tad, assign_bin
+
+plt.figure.max_open_warning = 0
+sns.set_style("whitegrid")
+sns.set_style("ticks")
+sns.set_context("paper", rc={"font.size": 20, "axes.titlesize": 20,
+                             "axes.labelsize": 20, "xtick.labelsize": 12,
+                             "ytick.labelsize": 12})
+
+
+def curate_gwas_elements(tad_df, input_df):
+    """
+    Loop through TAD boundaries to assign GWAS snps to TADs
+
+    Arguments:
+    :param tad_df: pandas dataframe, TAD boundary locations
+    :param input_df: pandas dataframe, the gwas locations
+
+    Output:
+    Large summary dataframes of the TAD assignments for GWAS signals
+    """
+
+    big_out_df = pd.DataFrame()
+
+    for tad in tad_df.itertuples():
+        tad_id, chrom, start, end = list(tad)
+        start, end = int(start), int(end)
+
+        chm_sub_df = input_df[input_df['chromosome'] == chrom]
+
+        elem_sub_df = chm_sub_df[(chm_sub_df['position'] >= start) &
+                                 (chm_sub_df['position'] < end)]
+
+        # Assign TAD information
+        elem_sub_df['TAD_id'] = tad_id
+        elem_sub_df['TAD_start'] = start
+        elem_sub_df['TAD_end'] = end
+
+        big_out_df = big_out_df.append(elem_sub_df, ignore_index=True)
+
+    return big_out_df
+
```

Comment:
yeah, it looks like the default plot arguments should be placed after the function - I'll update this in a new pull request since this one is merged already!